websites/B/Bungie/presence.ts

```diff
@@ -0,0 +1,160 @@
USxDQUFFLG1CQUFtQixDQUFDLEVBQUM7UUFDN0MsWUFBWSxDQUFDLGNBQWMsR0FBRyxhQUFhLENBQUM7UUFDNUMsWUFBWSxDQUFDLE9BQU8sR0FBRyxDQUFDLHNCQUFzQixDQUFDLENBQUE7S0FDOUM7U0FFSSxJQUFLLElBQUksQ0FBQyxRQUFRLENBQUUsaUJBQWlCLENBQUMsRUFBQztRQUMxQyxJQUFJLFFBQVEsR0FBRyxRQUFRLENBQUMsYUFBYSxDQUFDLFlBQVksQ0FBQyxDQUFBO1FBQ25ELFlBQVksQ0FBQyxjQUFjLEdBQUcsYUFBYSxDQUFDO1FBQzVDLFlBQVksQ0FBQyxPQUFPLEdBQUcsQ0FBQywwQkFBMEIsR0FBQyxRQUFRLENBQUMsU0FBUyxDQUFDLENBQUE7S0FDdkU7U0FFSSxJQUFLLElBQUksQ0FBQyxRQUFRLENBQUUseUJBQXlCLENBQUMsRUFBQztRQUNsRCxZQUFZLENBQUMsY0FBYyxHQUFHLGFBQWEsQ0FBQztRQUM1QyxZQUFZLENBQUMsT0FBTyxHQUFHLENBQUMsNkNBQTZDLENBQUMsQ0FBQTtLQUN2RTtTQUNJLElBQUssSUFBSSxDQUFDLFFBQVEsQ0FBRSxzQkFBc0IsQ0FBQyxFQUFDO1FBQ2hELElBQUksV0FBVyxHQUFHLFFBQVEsQ0FBQyxhQUFhLENBQUMsaUdBQWlHLENBQUMsQ0FBQTtRQUM1SSxJQUFJLFdBQVcsR0FBRyxRQUFRLENBQUMsYUFBYSxDQUFDLHFHQUFxRyxDQUFDLENBQUE7UUFDN0ksWUFBWSxDQUFDLGNBQWMsR0FBRyxhQUFhLENBQUM7UUFDNUMsWUFBWSxDQUFDLE9BQU8sR0FBRyxDQUFDLGFBQWEsR0FBQyxXQUFXLENBQUMsU0FBUyxHQUFHLE9BQU8sR0FBRSxXQUFXLENBQUMsU0FBUyxDQUFDLENBQUE7S0FHOUY7U0FFSSxJQUFJLElBQUksSUFBRyxVQUFVLEVBQUM7UUFDekIsWUFBWSxDQUFDLGNBQWMsR0FBRyxhQUFhLENBQUM7UUFDNUMsWUFBWSxDQUFDLE9BQU8sR0FBRyxDQUFDLDBCQUEwQixDQUFDLENBQUE7S0FDcEQ7U0FDSSxJQUFJLElBQUksSUFBRyxhQUFhLEVBQUM7UUFDNUIsWUFBWSxDQUFDLGNBQWMsR0FBRyxhQUFhLENBQUM7UUFDNUMsWUFBWSxDQUFDLE9BQU8sR0FBRyxDQUFDLDBCQUEwQixDQUFDLENBQUE7S0FDcEQ7U0FDSSxJQUFLLElBQUksQ0FBQyxRQUFRLENBQUUsZ0JBQWdCLENBQUMsRUFBQztRQUN6QyxJQUFJLFlBQVksR0FBRyxRQUFRLENBQUMsYUFBYSxDQUFDLGNBQWMsQ0FBQyxDQUFBO1FBQ3pELFlBQVksQ0FBQyxjQUFjLEdBQUcsYUFBYSxDQUFDO1FBQzVDLFlBQVksQ0FBQyxPQUFPLEdBQUcsQ0FBQyxVQUFVLEdBQUUsWUFBWSxDQUFDLFNBQVMsQ0FBQyxDQUFBO0tBQzVEO1NBQ0ksSUFBSSxJQUFJLENBQUMsUUFBUSxDQUFFLG1CQUFtQixDQUFDLEVBQUM7UUFDM0MsSUFBSSxjQUFjLEdBQUcsUUFBUSxDQUFDLGFBQWEsQ0FBQyxvTEFBb0wsQ0FBQyxDQUFBO1FBQ2pPLFlBQVksQ0FBQyxjQUFjLEdBQUcsYUFBYSxDQUFDO1FBQzVDLFlBQVksQ0FBQyxPQUFPLEdBQUcsQ0FBQyxVQUFVLEdBQUMsY0FBYyxDQUFDLFNBQVMsQ0FBQyxDQUFBO0tBQzdEO1NBQ0ksSUFBSSxJQUFJLENBQUMsUUFBUSxDQUFFLG9CQUFvQixDQUFDLEVBQUM7UUFDNUMsSUFBSSxhQUFhLEdBQUcsUUFBUSxDQUFDLGFBQWEsQ0FBQyxvQ0FBb0MsQ0FBQyxDQUFBO1FBQ2hGLFlBQVksQ0FBQyxjQUFjLEdBQUcsYUFBYSxDQUFDO1FBQzVDLFlBQVksQ0FBQyxPQUFPLEdBQUcsQ0FBQyxVQUFVLEdBQUMsYUFBYSxDQUFDLFNBQVMsQ0FBQyxDQUFBO0tBQzVEO1NBQ0ksSUFBSSxJQUFJLENBQUMsUUFBUSxDQUFFLHVCQUF1QixDQUFDLEVBQUM7UUFDaEQsSUFBSSxZQUFZLEdBQUcsUUFBUSxDQUFDLGFBQWEsQ0FBQyxtS0FBbUssQ0FBQyxDQUFBO1FBQzdNLFlBQVksQ0FBQyxjQUFjLEdBQUcsYUFBYSxDQUFDO1FBQzVDLFlBQVksQ0FBQyxPQUFPLEdBQUcsQ0FBQyxVQUFVLEdBQUMsWUFBWSxDQUFDLFNBQVMsQ0FBQyxDQUFBO0tBQzNEO1NBQ0ksSUFBSyxJQUFJLENBQUMsUUFBUSxDQUFFLDBCQUEwQixDQUFDLEVBQUM7UUFDbkQsSUFBSSxXQUFXLEdBQUcsUUFBUSxDQUFDLGFBQWEsQ0FBQyxtS0FBbUssQ0FBQyxDQUFBO1FBQy9NLFlBQVksQ0FBQyxjQUFjLEdBQUcsYUFBYSxDQUFDO1FBQzVDLFlBQVksQ0FBQyxPQUFPLEdBQUcsQ0FBQyxVQUFVLEdBQUMsV0FBVyxDQUFDLFNBQVMsQ0FBQyxDQUFBO0tBQ3hEO0lBQ0QsSUFBSSxZQUFZLENBQUMsT0FBTyxJQUFJLElBQUksRUFBRTtRQUNoQyxRQUFRLENBQUMsWUFBWSxFQUFFLENBQUM7UUFDeEIsUUFBUSxDQUFDLFdBQVcsRUFBRSxDQUFDO0tBQ3hCO1NBQ0k7UUFDSCxRQUFRLENBQUMsV0FBVyxDQUFDLFlBQVksQ0FBQyxDQUFDO0tBQ3BDO0FBQUEsQ0FBQyxDQUFDLENBQUEifQ==
```

Comment:
there is in no way possible to have a sourcemapping in a ts file. So funny how you manage to do this.

Reply:
Guess what, it just did it on it's own.